Mono.Cecil.Cil/Symbols.cs

```diff
@@ -1224,5 +1226,38 @@ public static bool IsPortablePdb (Stream stream)
 				stream.Position = position;
 			}
 		}
+
+		public static bool GetHasCustomDebugInformations (
+			this ICustomDebugInformationProvider self,
+			ref Collection<CustomDebugInformation> collection,
+			ModuleDefinition module)
+		{
+			if (module.HasImage ())
+				module.Read (ref collection, self, (provider, reader) => {
+					var symbol_reader = reader.module.symbol_reader;
+					if (symbol_reader != null)
+						return symbol_reader.Read (provider);
+					return null;
+				});
```

Comment:
nit: for multiline/single statement `if`s I find it easier to read with brackets

```suggestion
			if (module.HasImage ()) {
				module.Read (ref collection, self, (provider, reader) => {
					var symbol_reader = reader.module.symbol_reader;
					if (symbol_reader != null)
						return symbol_reader.Read (provider);
					return null;
				});
			}
```